netebpfext/net_ebpf_ext.c

```diff
@@ -33,10 +33,12 @@
 #include <netiodef.h>
 #include <ntddk.h>
 
+#include "ebpf_bind_program_data.h"
```

Comment:
I don't see this file or ebpf_xdp_program_data.h in this PR, where is it?

Reply:
It's generated by encode_program_information.

Problem:
We can't call RPC to encode the type information from the kernel.

Solution:
1) Call RPC from user mode.
2) Store the resulting buffer in header file.
3) Use encoded buffer from driver.

Given that the program information is defined at compile time, we can get away with it.

We still need to solve serialization for the IOCTL path, but this should unblock the verifier changes.

Reply:
Can you add this explanation as a // comment somewhere in the code?